renderer.js

```diff
@@ -985,14 +1010,85 @@ function playPressed(){
 
         playModalQuitDLCancel = dataObj.reqObj;
         
+        
     });
 
 }
 
-playButtonText.addEventListener("click", function(event){
+async function checkIfCompatible() {
+    try {
+        const data = await si.graphics();
 
+        // Get all of the graphics cards info
+        var i;
+        for (i = 0; i < data.controllers.length; i++){
+            graphicControllers = data.controllers[i];
+            cards = String(graphicControllers.model);
+            console.log(cards);
+        }
+
+        // Is incompatible if the word Intel is found in a substring
+        if(cards.includes("Intel")){
+            showIncompatiblePopup = true;
+
+            // Show help text
+            if(cards.includes("Nvidia") || cards.includes("AMD")){
+                showHelpText = true;
+            }
+        }
+
+    } catch (e) {
+        console.log(e)
+    }
+}
+
+playButtonText.addEventListener("click", function(event){
     console.log("play clicked");
-    playPressed();
+
+    if(showIncompatiblePopup){
+        incompatibleModal.show();
+
+        let incompatibleBox = document.getElementById("incompatibleModalContent");
+        incompatibleBox.innerHTML = "<p id='text'></p>"
+    
+        let box = document.getElementById("text");
+
+        box.textContent = "Detected graphics card(s): " + cards;
+
+        if(showHelpText){
+            box.append(document.createElement("br"));
+            box.append(document.createElement("br"));
+            box.append(document.createTextNode("Another graphics card detected, you could configure Thrive to run with that instead!"));
+        }
+        box.append(document.createElement("br"));
+        box.append(document.createElement("br"));
+        box.append(document.createTextNode("WARNING: Intel Integrated Graphics card may causes Thrive to crash due to issues with the graphics engine running on it: https://github.com/Revolutionary-Games/Thrive/issues/804."))
+        box.append(document.createElement("br"));
+        box.append(document.createTextNode("This is a known problem, any help fixing this would be very much appreciated!"));
+        
+        let closeContainer = document.createElement("div");
+        closeContainer.style.marginTop = "20px";
+        closeContainer.style.textAlign = "center";
+        let close = document.createElement("div");
+        close.textContent = "Continue";
+        close.className = "CloseButton";
+    
+        close.addEventListener('click', (event) => {
+            incompatibleModal.hide();
+            if(cards != null){
+                playPressed();
```

Comment:
Maybe `showIncompatiblePopup` should be set to false here to only show the box once?
We can later add an option to permanently dismiss the popup.

Reply:
This still seems to be the case...

Reply:
Still not fixed.